src/database/models/eventsModel.js

```diff
@@ -2,34 +2,37 @@ var mongoose = require('mongoose');
 var Schema = mongoose.Schema;
 
 //Event model consisting of the event title, description, start and end time, checkIn code, and splash URL
-var eventSchema = new Schema(
-    {
-    title: {
-        type: String,
-        required: true
-    },
-    location: {
-        type: String,
-        required: true
-    },
-    startTime: {
-        type: Date,
-        required: true
-    },
-    endTime: {
-        type: Date,
-        required: true
-    },
-    checkIn: {
-        type: String,
-        required: true
-    },
-    url: {
-        type: String,
-        required: false
-    }
-})
+var eventSchema = new Schema({
+  title: {
+    type: String,
+    required: true
+  },
+  location: {
+    type: String,
+    required: true
+  },
+  startTime: {
+    type: Date,
+    required: true
+  },
+  endTime: {
+    type: Date,
+    required: true
+  },
+  checkIn: {
+    type: String,
+    required: true
+  },
+  tags: {
```

Comment:
why are the tags here a string? what if we assign multiple to an event?

Reply:
Lol I was thinking of using Array but I feel like that might cause formatting issues when people try to add tags so I was thinking of having a string of tags separated by commas but @brilam8 @garrettluu have more experience what do u guys think?

Reply:
Hmm what formatting issues are you concerned about? I think that doing a string separated by commas is more work since we're gonna have to parse it at some point if we are using these tags as a way to search for events.

Reply:
agreed with Garrett, I guess either way works but an array would probably be easiest to work with and parse through when we need it